app/forms.py

```diff
@@ -153,41 +153,54 @@ class EditProfileForm(RegistrationForm):
         label="Baby has been born",
         required=False
     )
+    image = forms.ImageField()
+    sur_name = forms.CharField(
+        max_length=100,
+        label="Surname",
+        required=False
+    )
+    engage_anonymously = forms.BooleanField(
+        label="Engage Anonymously",
+        required=False
+    )
 
     def __init__(self, *args, **kwargs):
         super(EditProfileForm, self).__init__(*args, **kwargs)
-        self.fields['date_qualifier'].widget = forms.HiddenInput()
-        del self.fields['due_date']
+        #self.fields['date_qualifier'].widget = forms.HiddenInput()
+        #del self.fields['due_date']
         self.fields.keyOrder = [
             'username',
             'mobile_number',
-            'relation_to_baby',
+            # 'relation_to_baby',
             'about_me',
             'baby_name',
-            'date_qualifier',
-            'delivery_date',
-            'unknown_date',
-            'baby_has_been_born'
+            # 'date_qualifier',
+            # 'delivery_date',
+            # 'unknown_date',
+            'baby_has_been_born',
```

Comment:
This can also be removed. Please also just delete the fields instead of commenting them out.